pkg/conference/participant/track.go

```diff
@@ -0,0 +1,100 @@
+package participant
+
+import (
+	"time"
+
+	"github.com/matrix-org/waterfall/pkg/peer"
+	"github.com/thoas/go-funk"
+)
+
+type TrackID = string
+
+// Represents a track that a peer has published (has already started sending to the SFU).
+type PublishedTrack struct {
+	// Owner of a published track.
+	Owner ID
+	// Info about the track.
+	Info peer.TrackInfo
+	// Available simulcast Layers.
+	Layers []peer.SimulcastLayer
+	// Track metadata.
+	Metadata TrackMetadata
+	// The timestamp at which we are allowed to send the FIR or PLI request. We don't want to send them
+	// too often, so we introduce some trivial rate limiting to not "enforce" too many key frames.
+	canSendKeyframeAt time.Time
```

Comment:
```suggestion
	canRequestKeyframeAt time.Time
```

Reply:
NB: I just took the name of this field from the previous implementation 🙂 Though I can definitely rename it, requesting probably reflects its meaning better.

Reply:
Done.